pages/edge/quickstart/python.mdx

```diff
@@ -0,0 +1,100 @@
+import { Steps, Callout, FileTree } from "nextra-theme-docs";
+import Install from "@components/install.mdx";
+import Login from "@components/login.mdx";
+import CliVersionCallout from "@components/deploy/CliVersionCallout.mdx";
+
+# Quickstart guide for deploying a Python Application
+
+In this guide, you'll learn the process of deploying a python application on Wasmer Edge.
+We'll be deploying a basic http server written in python.
+
+## Deploying a Python Application
+
+<Steps>
+
+<Install />
+
+<CliVersionCallout />
+
+<Login />
+
+### Step 3.1: Initialize the Python Application Starter template
+
+```shell copy
+$ wasmer app create
+App type:
+  Static website
+  HTTP server
+  Browser shell
+  JS Worker (experimental)
+> Python Worker
```

Comment:
```suggestion
> Python Webserver
```

Reply:
It's in the CLI now. Nothing can be done till the next wasmer release